dashboard/app/controllers/lti/v1/account_linking_controller.rb

```diff
@@ -0,0 +1,36 @@
+module Lti
+  module V1
+    class AccountLinkingController < ApplicationController
+      before_action :lti_account_linking_enabled?
+
+      # GET /lti/v1/account_linking/landing
+      def landing
+        render 'lti/v1/account_linking/landing_page'
```

Comment:
Please rename the template to match the action name (`views/lti/v1/account_linking/landing.html.haml`) and then you can remove the render call
```suggestion
```